src/Commands/Env/CommitCommand.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Env;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Terminus\Exceptions\TerminusException;
+use Terminus\Collections\Sites;
+
+class CommitCommand extends TerminusCommand
+{
+    /**
+     * Contructor that supports injection for unit testing.
+     */
+    public function __construct($sites = null)
+    {
+        parent::__construct();
+        $this->sites = $sites ? $sites : new Sites();
+    }
+
+    /**
+     * Commit code on an environment that is in SFTP mode.
+     *
+     * @command env:commit
+     *
+     * @param string $site_env Site & environment to commit code on.
+     *
+     * @option string $message Commit message
+     *
+     * @usage terminus env:commit my-site.dev --message="My code changes"
+     *   Commit changes to the `dev` environment for site `my-site`.
+     */
+    public function commit($site_env, $options = ['message' => 'Terminus commit.'])
+    {
+        list($site_name, $env_name) = explode('.', $site_env);
+        $site = $this->sites->get($site_name);
+        $env  = $site->environments->get($env_name);
+
+        // TODO: The 0.x command would prompt if there were no changed files, asking
+        //   if the user wanted to commit anyway.
```

Comment:
I can't tell why the 0.x command prompts in this way. For now, let's just go with:

 "[warning] There is no code to commit on <site.env>." 
or 
 "[warning] There is no code to commit." 

like
<img width="499" alt="screenshot 2016-09-29 01 47 07" src="https://cloud.githubusercontent.com/assets/7874556/18947134/ad277c96-85e6-11e6-977b-e0ba4ccf6acf.png">
."